astroquery/eso/core.py

```diff
@@ -207,6 +207,8 @@ def list_instruments(self, cache=True):
                     if instrument not in self._instrument_list:
                         self._instrument_list.append(instrument)
             self._instrument_list.append(u'harps')
+            self._instrument_list.append(u'ambient_paranal')
+            self._instrument_list.append(u'meteo_paranal')
```

Comment:
Maybe because `harps` is at La Silla, and because `ambient_paranal` and `meteo_paranal` are not really instruments? We could ask `archive_AT_eso.org` to add them there...